bytecode-compiler/bytecompiler.rewrite.md

```diff
@@ -1282,6 +1255,12 @@ For these (figure 2), we'll need more passes:
          | ListComp(expr elt, comprehension* generators)
     comprehension = (expr target, expr iter, expr* ifs)
 
+Once again, we ask ourselves if our current design can handle the
+complexity we've added in this version. 
+
+DEBO NOTE: [We need some kind of bridge here to demonstrate why (a) the answer is
+now, and (b) why we need 'desugaring' and 'scope analysis'.]
```

Comment:
I didn't feel comfortable enough with the source material to provide this bridge.